### DISCUSSION TITLE
Carto map features

### DISCUSSION THREAD

**Discussion by veryphatic** [TerriaJS/terriajs#6727] [General] [answered]
Hi there, I've consumed some CartoDB data with the 'carto' item, using data from the [NSW  restricted vehicle map](https://roads-waterways.transport.nsw.gov.au/business-industry/heavy-vehicles/maps/restricted-access-vehicles-map/map/index.html). The API queries retuning the imagery; however, when clicking on the map the feature information modal indicates _No data available_. 

I'm wondering if either I've missed something, or the feature isn't implemented?

I I'm running TerriaMaps 0.0.6.

**Comment by imakihi** [comment, marked as answer]
@veryphatic 
Hi there. Yes, I also could not fetch attribute values by clicking features. We ended up developing this function on our own. I will ask our developer how he implemented, if you are interested in.

**Reply by nf-s** [reply]
Yes @imakihi is right, we unfortunately don't have support for feature picking with the Carto Map API v1. At this stage I think it is unlikely we will add support as the API will soon become deprecated.

> This component is still under support but it will not be further developed. We don’t recommend starting new projects with it as it will eventually become deprecated. Instead, learn more about our new APIs [here](https://api-docs.carto.com/)

https://carto.com/developers/maps-api/reference/
